packages/grafana-ui/src/components/ContextMenu/ContextMenu.story.tsx

```diff
@@ -24,14 +24,17 @@ const menuItems = [
     items: [
       { label: 'First', ariaLabel: 'First' },
       { label: 'Second', ariaLabel: 'Second' },
+      { label: 'Third', ariaLabel: 'Third' },
+      { label: 'Fourth', ariaLabel: 'Fourth' },
+      { label: 'Fifth', ariaLabel: 'Fifth' },
     ],
   },
 ];
 
 const renderMenuItems = () => {
   return menuItems?.map((group, index) => (
     <MenuGroup key={`${group.label}${index}`} label={group.label} ariaLabel={group.label}>
-      {(group.items || []).map((item) => (
+      {(group.items ?? []).map((item) => (
```

Comment:
Wonder why this change? In this case if `group.items` somehow ends being an empty string or `0` the code will throw an error. Also I think `group.items?.map` should work here.

Reply:
+1 for `group.items?.map`, although that will still have the same problem if `group.items` is somehow `''` or `0`. i think that's extremely unlikely though. typescript should be stopping that from happening.